Server/BackEndMonitoringServer/src/CLogicalDiskStatusLifeCycle.cpp

```diff
@@ -0,0 +1,55 @@
+#include "stdafx.h"
+
+#include "CLogicalDiskStatusLifeCycle.h"
```

Comment:
The same as above

Reply:
If you talking abut moving includes to *.cpp - fixed in [commit](https://github.com/ITA-Dnipro/BackEndMonitoring/commit/28775d4e4e211a6985cecfb05174a510800a77d4)